Directory.Build.props

```diff
@@ -230,9 +230,11 @@
     <RepositoryUrl>https://github.com/dotnet/$(GitHubRepositoryName)</RepositoryUrl>
     <PackageProjectUrl>https://dot.net</PackageProjectUrl>
     <Owners>microsoft,dotnetframework</Owners>
+    <IncludeSymbols>true</IncludeSymbols>
```

Comment:
Does this global setting effect other packages? We do have a few others that are already NuGet Pack task based. What about the existing pkgproj ones?

Reply:
> What about the existing pkgproj ones, especially coreclr, mono and installer ones.

They don't use this property. They have their own set of properties.


> Does this global setting affect other packages? We do have a few others that are already NuGet Pack task based.

i restored them to the state they were before this change, for some of them we might want to revisit on why they are not publishing symbols.

Reply:
Thanks for double checking. As I'm still a bit worried about this, cc'ing @ericstj in case he has more opinions on this change.